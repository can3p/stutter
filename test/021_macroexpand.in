;; Test multi-level macro expand.

(defmacro inner-macro (arg) `(list "hello" ,arg))

(defmacro top-level-macro (arg) `(inner-macro ,arg))

(list
 (inner-macro "world")
 (top-level-macro "universe")
 (macroexpand '(top-level-macro "amsterdam")))
