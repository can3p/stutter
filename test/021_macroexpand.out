(("hello" "world") ("hello" "universe") (list "hello" "amsterdam"))
